src/components/tooltip/Tooltip.tsx

```diff
@@ -142,68 +134,79 @@ const Tooltip: React.FC<TooltipProps> = ({
     ],
   });
 
-  const children = <span>{otherProps.children}</span>;
-
   const isVisible =
     typeof visible !== 'undefined'
       ? visible
       : displayTrigger === 'hover'
         ? showHover
         : showClick;
 
+  const children = <div
+    className={ classnames('tk-tooltip__wrapper', wrapperClassName) }
+    ref={ composeRefs(setReferenceElement, ref) }
+  >
+    {otherProps.children}
+  </div>
+
   return (
-    <div ref={ref} className={className ? className + '_wrapper' : null}>
-      <SpanStyled ref={setReferenceElement}>
-        {displayTrigger === 'hover' &&
+    <>
+      { /**
+       * The element tooltip is wrapped around.
+       */ }
+      { /** Extra span to make sure Tooltip works on disabled children */ }
```

Comment:
Where is the span?

Reply:
Moved the comment up to relevant section. Thanks!